testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -4947,6 +4947,22 @@ QUnit.module('aria accessibility', {}, () => {
             assert.ok(true, 'skip test on devices');
         }
     });
+
+    QUnit.test('aria-controls should be set if the popup has been rendered', function(assert) {
+        const isDesktop = devices.real().deviceType === 'desktop';
+
+        if(isDesktop) {
+            const $element = $('#dateBox').dxDateBox({ value: new Date() });
+            const $input = $element.find(`.${TEXTEDITOR_INPUT_CLASS}`);
+
+            const $button = $element.find(`.${DROP_DOWN_BUTTON_CLASS}`);
+            $($button).trigger('dxclick');
```

Comment:
why not just use [open()](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDateBox/Methods/#open) method here? And `field()` to get input element